card-flow.md

```diff
@@ -0,0 +1,52 @@
+
+#### The "Next" Column
+Once cards are approved and ready for developers to begin working on them, we move them to the "Next" column. Cards in this column are our queue of work, and are waiting for someone to be assigned to them. A developer can assign themselves to a card when they are ready to begin work on it.
+
+#### The "Development" Column
+
+The "Development" column is for cards that are actively being worked on. It is important that a developer has only one card in this column at a time. If they need to work on something else, this card can move back to ready so as not to let things build up or block. Our project board system follows the traditional Kanban pull based system pretty closely. Things can only move right if there's space for them, and a dev team can really only take on 3-4 cards at once in the development stage. More than that and things sit idle. The board should be an accurate reflection of the project at any point in time, so if a developer is not currently working on a card in the "Development" column, they can move it back so our progress is accurately reflected.
+
+#### The "Needs Review" Column
+
+When a developer has completed the task on a card, we kick off the (pull request process)[https://github.com/HaughtCodeworks/TeamGuide]. The "Needs Review" column represents cards currently under code review in the PR process. After a PR has been fully approved, the feature is merged into the master branch and deployed to the staging environment.
+
+#### The "Staging" Column
+
+Cards in the "Staging" column are ready for client review on the staging environment. The client makes sure everything on the feature works as expected.
+
+#### The "Approved" Column
+
+Once clients have tried out a feature and approved a card in "Staging", they move the card to the "Approved" column.
+
+#### The "Live" Column
+
+After the client moves a card to "Approved", a developer will deploy the app to the live environment. We verify it internally, and the client needs to approve and sign off on the feature. At this point, the task card is considered complete. We close the task card, and move it from the board.
+
+## Cards
+
+Cards each represent a specific component of a project, such as a user story, bug fix, development or design task, or a general to-do. The tasks on a card should match an ideal pull request size so that the card can follow a pull request as it goes through. A card that includes too many tasks would lead to multiple pull requests on the same card. If you find this is the case, split a task into multiple cards.
```

Comment:
A slight tweak here.  Let's try something like this:

Cards represent a request for work such as a new feature, a user story, bug fix, design modification or anything else that the team needs to perform while having its progressed tracked by the client.
A single card should contain a reasonable size of work that should be completed and shipped together.  Though we may break down a card into tasks via a checklist, ideally all should fit into a single pull request and move across the board as one.  A card that includes too many tasks would lead to multiple pull requests on the same card. If you find this is the case, split a task into multiple cards.